docs/docs/genai/eval-monitor/scorers/llm-judge/migration-guide.mdx

```diff
@@ -0,0 +1,570 @@
+import TabItem from "@theme/TabItem";
+import TileCard from "@site/src/components/TileCard";
+import TilesGrid from "@site/src/components/TilesGrid";
+import { MessageSquare, BookOpen, Rocket } from "lucide-react";
+
+# Migrating from Legacy Judges to make_judge
+
+:::warning[Deprecation Notice]
+
+The legacy judge functions (`is_context_relevant`, `is_correct`, `is_grounded`, `is_context_sufficient`, `meets_guidelines`, and `custom_prompt_judge`) are deprecated as of MLflow 3.4.0 and will be removed in a future release. This guide helps you migrate to the new, more flexible `make_judge` API.
+
+:::
+
+## Why Migrate?
+
+The new <APILink fn="mlflow.genai.judges.make_judge">make_judge</APILink> API provides significant improvements over the legacy judge functions:
+
+- **Unified Interface**: Single API for all judge types instead of multiple functions
+- **Greater Flexibility**: Support for arbitrary inputs, outputs, and expectations
+- **Future-Proof**: Will support traces, tool calling, and alignment features
+- **Better Extensibility**: Easier to create custom evaluation criteria
+- **Consistent Experience**: Works seamlessly with MLflow scorers
+
+## Migration Examples
+
+The following examples show how to migrate from each deprecated judge function to the new `make_judge` API. Note that `is_safe` is not included as it is specific to Databricks environments.
+
+### Migrating `is_context_relevant`
+
+<Tabs>
+  <TabItem value="legacy" label="Legacy Approach" default>
+    ```python
+    from mlflow.genai.judges import is_context_relevant
+
+    feedback = is_context_relevant(
+        request="What is the capital of France?",
+        context="Paris is the capital of France.",
+        model="openai:/gpt-4",
+    )
+    ```
+
+  </TabItem>
+  <TabItem value="new" label="New Approach">
+    ```python
+    from mlflow.genai.judges import make_judge
+
+    # Create a reusable judge
+    context_relevance_judge = make_judge(
+        name="context_relevance",
+        instructions="Is the context {{context}} relevant to answering the request {{request}}? Answer yes or no.",
+        model="openai:/gpt-4",
+    )
+
+    # Use the judge
+    feedback = context_relevance_judge(
+        inputs={"request": "What is the capital of France?"},
+        expectations={"context": "Paris is the capital of France."},
+    )
+    ```
+
+  </TabItem>
+</Tabs>
+
+### Migrating `is_correct`
+
+<Tabs>
+  <TabItem value="legacy" label="Legacy Approach" default>
+    ```python
+    from mlflow.genai.judges import is_correct
+
+    feedback = is_correct(
+        request="What is 2+2?", response="4", expected_response="4", model="openai:/gpt-4"
+    )
+    ```
+
+  </TabItem>
+  <TabItem value="new" label="New Approach">
+    ```python
+    from mlflow.genai.judges import make_judge
+
+    correctness_judge = make_judge(
+        name="correctness",
+        instructions="""
+        Evaluate if the response {{response}} correctly answers the request {{request}}.
```

Comment:
@BenWilson2 we need to update these examples based on the new contract for the input variables in`make_judge`.